resources/js/interfaces/score-json.ts

```diff
@@ -21,7 +21,7 @@ interface ScoreCurrentUserAttributesJson {
 interface Match {
   pass: boolean;
   slot: number;
-  team: number;
+  team: 'blue' | 'none' | 'red';
```

Comment:
Rationale for this change is that in the actual serialised json this appears to be a string, probably thanks to

https://github.com/ppy/osu-web/blob/026c39d05218ce2034693c46bceab7479895acc9/app/Models/LegacyMatch/Score.php#L87-L90

I wasn't sure whether I should be changing the attribute definition on the php model because it's confusing (in `artisan tinker` when I fetch the entire model the object dump shows an integer inside, but when I access the actual property it magically stringifies itself) so I didn't.

Reply:
I made an attempt in https://github.com/ppy/osu-web/pull/12030/commits/32b1e29c6d7337eb1482d868c59b6b3aaffd28b7, although I'm not sure as to whether you were expecting it to change to `string` or `mixed` (I did the latter, because of its weird stringy-numbery duality as described above).